backends/beats/filebeat/render.go

```diff
@@ -198,3 +204,64 @@ func (fbc *FileBeatConfig) ValidateConfigurationFile() bool {
 
 	return true
 }
+
+func (fbc *FileBeatConfig) runMigrations(bc *beats.BeatsConfig) {
```

Comment:
I think it would be good if we create some tests for the migration. This can break pretty fast.